src/code/z_fbdemo.c

```diff
@@ -45,11 +58,13 @@ void TransitionUnk_InitGraphics(TransitionUnk* this) {
     for (frame = 0; frame < 2; frame++) {
         this->frame = frame;
         vtx = (this->frame == 0) ? this->vtxFrame1 : this->vtxFrame2;
-        for (colTex = 0, col = 0; col < this->col + 1; colTex += 0x20, col++) {
-            for (rowTex = 0, row = 0; row < this->row + 1; row++) {
-                vtx2 = &vtx->v;
-                vtx++;
+        colTex = 0;
+        for (col = 0; col < this->cols + 1; col++) {
+            rowTex = 0;
+            for (row = 0; row < this->rows + 1; row++) {
+                Vtx_t* vtx2 = &vtx->v;
```

Comment:
While you're here, this should use `Vtx_tn` since these vertices end up drawn using the settings in `sTransTileSetupDL`. which sets `G_LIGHTING` in the geometrymode